### PR TITLE
Feature/bar chart

### PR DESCRIPTION
**Pull request by namhai923**

### CODE REVIEW

File: .github/workflows/deploy.yml
Comment (LimitlessJesse) [inline .github/workflows/deploy.yml:1]: oops

File: back-end/Dockerfile
Comment (LimitlessJesse) [inline back-end/Dockerfile:1]: oops

**Review by LimitlessJesse — APPROVED** [review]
Looks good

**Review by afazmunshi49 — APPROVED** [review]
lgtm

**Pull request merged**